#!/usr/bin/env python
PACKAGE = "hector_heat_detection"
import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("min_temp_detection", double_t, 0, "Minimum detection temperature", 20.0, -20.0, 100.0)
gen.add("max_temp_detection", double_t, 0, "Maximum detection temperature", 23.0, -20.0, 100.0)
gen.add("min_area_detection", double_t, 0, "Minimum detection area", 40.0, 0.0, 10000.0)
gen.add("min_dist_between_blobs", double_t, 0, "Minimum distance between detections", 20.0, 0, 1000.0)
gen.add("percept_class_id", str_t, 0, "The percept class string", "victim")

exit(gen.generate(PACKAGE, "heat_detection", "HeatDetection"))
